README.md

```diff
@@ -786,8 +786,8 @@ task jes_task {
     docker: "ubuntu:latest"
     memory: "4G"
     cpu: "3"
-    defaultZones: "US_Metro MX_Metro"
-    defaultDisks: "Disk1 3 SSD, Disk2 500 HDD"
+    zones: "US_Metro MX_Metro"
+    disks: "Disk1 3 SSD, Disk2 500 HDD"
```

Comment:
I went to green's stand-up this morning. Yes, I will do this but I'll leave it up to them when to merge.